dockerfiles/alpine/Dockerfile

```diff
@@ -75,7 +75,7 @@ RUN apk update \
   && adduser --shell /bin/bash -G bobd -G wheel -S bobd
 
 WORKDIR /home/bob/bin/
-COPY --from=cargo-build /usr/src/bob/target/x86_64-unknown-linux-musl/release/bobd .
+COPY --from=cargo-build /usr/src/bob/target/x86_64-unknown-linux-musl/${BUILD_PROFILE_DIR:-"release"}/bobd .
 RUN chown bobd:bobd bobd \
```

Comment:
Please fix this line: do `chown` on `bobd` within `COPY` (`COPY --chown=bobd:bobd --from=cargo-build /usr/src/bob/target/x86_64-unknown-linux-musl/release/bobd`)